hudi-common/src/main/java/org/apache/hudi/common/table/timeline/dto/FileGroupDTO.java

```diff
@@ -50,13 +50,13 @@ public static FileGroupDTO fromFileGroup(HoodieFileGroup fileGroup) {
     dto.partition = fileGroup.getPartitionPath();
     dto.id = fileGroup.getFileGroupId().getFileId();
     dto.slices = fileGroup.getAllRawFileSlices().map(FileSliceDTO::fromFileSlice).collect(Collectors.toList());
-    dto.timeline = TimelineDTO.fromTimeline(fileGroup.getTimeline());
+    dto.timeline = TimelineDTO.fromTimeline(fileGroup.getVisibleActiveCompletedTimeline());
```

Comment:
may be we should store entire timeline in DTO (complete write timeline).
while constructing the FileGroup from DTO object, we can split it into complete and entire write timeline. wdyt